src/lib.rs

```diff
@@ -3969,6 +3983,121 @@ export function a(a: A): B {
     }
   }
 
+  #[tokio::test]
```

Comment:
Would be good to add a test for where there's a `type` entry and an import has a `deno-types` pragma (the pragma should win).